cocotb/triggers.py

```diff
@@ -66,6 +66,10 @@ def __del__(self):
     def __str__(self):
         return self.__class__.__name__
 
+    @property
+    def outcome(self):
+        return outcomes.Value(self)
```

Comment:
If you'd prefer, I could rename `Trigger.outcome` to `Trigger._outcome`, marking it as private for use only by the scheduler